src/main/java/org/opentripplanner/middleware/models/ApiKey.java

```diff
@@ -0,0 +1,72 @@
+package org.opentripplanner.middleware.models;
+
+import com.amazonaws.services.apigateway.model.CreateApiKeyResult;
+import com.amazonaws.services.apigateway.model.GetApiKeyResult;
+
+import java.util.Objects;
+
+/**
+ * Represents a subset of an AWS API Gateway API key.
+ */
+public class ApiKey {
+
+    /**
+     * The api key id as provided by AWS API Gateway.
+     */
+    public String id;
+
+    /**
+     * The name given to the api key.
+     */
+    public String name;
+
+    /**
+     * The api key value. Can not be transient as it is excluded from the Spark response.
```

Comment:
@br648, I think we'll end up storing the API key value at some point by accident if we don't make this transient. Maybe we should just store the value and not hide it (if we do this, we could also remove the getApiKey endpoint/method). Ultimately I'm not really sure if there's much gained by keeping it opaque/out of the db (this was just my initial instinct). Curious to hear other's thoughts.

Reply:
Holding it in Mongo will save a trip to AWS. I'll wait feedback before updating this section.

Reply:
I defer to @evansiroky.

Reply:
Yeah, I think it makes sense to store all this in the db and not make so many get API key requests to AWS.

Reply:
Updated to save value in DB and removed getApiKey endpoint.